context.go

```diff
@@ -103,6 +104,12 @@ func (dc *Context) SavePNG(path string) error {
 	return SavePNG(path, dc.im)
 }
 
+// EncodePNG encodes the image as a PNG and
+// writes it to the provided io.Writer.
+func (dc *Context) EncodePNG(w io.Writer) error {
+	return EncodePNG(w, dc.im)
```

Comment:
This could just be `png.Encode(w, dc.im)` and then we don't even need the helper function in util.go.